src/main/java/org/opensearch/flowframework/rest/RestGetWorkflowAction.java

```diff
@@ -55,6 +55,11 @@ public String getName() {
         return GET_WORKFLOW_ACTION;
     }
 
+    @Override
```

Comment:
No problem moving this up but I'd like all our REST APIs to follow the same ordering

Reply:
I think the diff is wonky due to the file name changes. Essentially this new API `RestGetWorkflowTransportAction` is used to retrieve a template by workflow_id